### ISSUE TITLE
Update not working despite saying successful

### ISSUE THREAD

**Issue by Nicolas2705** [wenyi0421/turing-pi#16]
When I update the firmeware to the latest release following the documentation using the web ui, I receive a response with the result ok, however afterwards the firmeware is not updated. 
![Screenshot](https://user-images.githubusercontent.com/92875416/235163348-3df4c2e1-883e-4bfd-93a5-2228b70b3b93.PNG)

**Comment by charlieporth1** [comment]
I'm having this issue as well. Using custom forks in my case and it still is not updating

**Comment by wenyi0421** [comment]
Please check whether the device version is up to date. V1.0.0 has a problem that the upgrade file is too large, which may cause the upgrade to fail. You can upgrade the .img of V1.0.1 to fix this problem.

**Comment by nerocide** [comment]
Upgrade to v1.0.1 worked fine for me.

![image](https://user-images.githubusercontent.com/1773222/236200489-5a3e0c62-2258-43b5-bb7a-ecd55a4ef31f.png)

**Comment by drgncabe** [comment]
Did you pull power from the board post upgrade? I had to do this for the update to take effect. Even hitting the "reset BMC" button didn't do it.

**Comment by charlieporth1** [comment]
@drgncabe Yes I have. I've modified the version number for it to be above 1.0.2 as well to make sure I had changes. I modified the `/tmp` dir to be 128mb as well
